go/teams/ftl.go

```diff
@@ -1359,6 +1376,19 @@ func (f *FastTeamChainLoader) refresh(m libkb.MetaContext, arg fastLoadArg, stat
 
 	if groceries == nil {
 		m.Debug("FastTeamChainLoader#refresh: our state was fresh according to the Merkle tree")
+
+		// Even if the state is fresh (no new chain links are necessary), we
+		// might still have learned about old hidden links being committed to
+		// the blind tree. This information needs to be persisted.
+		err = hp.Update(m, []sig3.ExportJSON{}, 0)
+		if err != nil {
+			return nil, err
+		}
+		err = hp.Commit(m)
```

Comment:
Will this result in a write every time we call through here? It should be the case if no writes are needed, the data isn't rewritten. Since writes can be slow on mobile devices.

Reply:
from reading the code, it seems to me that is not the case: the `HiddenTeamChain` `merge` method returns a bool on whether there were any updates which should be written to storage. If you want me to be absolutely certain, I can write a test, but I am more confident than not that this is ok.